contracts/RFQ.sol

```diff
@@ -28,9 +30,23 @@ contract RFQ is IRFQ, StrategyBase, ReentrancyGuard, SignatureValidator, BaseLib
         bytes32 transactionHash;
     }
 
-    // Operator events
+    /// @notice Emitted when fee collector address is updated
+    /// @param newFeeCollector The address of the new fee collector
     event SetFeeCollector(address newFeeCollector);
 
+    /// @notice Emitted when order is filled
+    /// @param source The tag of the contract where the order is filled
+    /// @param transactionHash The hash of the transaction structure which is signed by taker
+    /// @param orderHash The hash of the order structure which is signed by maker
+    /// @param userAddr The address of taker
+    /// @param takerAssetAddr The taker assest used to swap
+    /// @param takerAssetAmount the swap amount of taker asset
+    /// @param makerAddr The address of maker
+    /// @param makerAssetAddr The maker assest used to swap
+    /// @param makerAssetAmount the swap amount of maker asset
+    /// @param receiverAddr The address of who receives the maker asset (always equals to the taker address)
```

Comment:
I think receiver could be assigned to different address other than taker address at client's will?

Reply:
just remove `(always equals to the taker address)`?

Reply:
Yes~~